app/src/main/java/com/framgia/forder/screen/shopmanagement/ShopManagementViewModel.java

```diff
@@ -1,14 +1,31 @@
 package com.framgia.forder.screen.shopmanagement;
 
+import android.databinding.Bindable;
```

Comment:
ERROR: Unused import: android.databinding.Bindable. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports